examples/golang-push/migrating-from-standard-pprof/README.md

```diff
@@ -0,0 +1,45 @@
+# Migrating from Standard pprof to Pyroscope in a Go Application
+## Introduction
```

Comment:
In this case, we don't need an introduction heading since it's the first paragraph -- which is usually the introduction. 
```suggestion
```